pkg/build/api/validation/validation.go

```diff
@@ -500,11 +500,14 @@ func validateTrigger(trigger *buildapi.BuildTriggerPolicy, buildFrom *kapi.Objec
 	return allErrs
 }
 
-func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path) field.ErrorList {
+func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path, allowEnv bool) field.ErrorList {
```

Comment:
Note that once you're done here with whether it's allowed, oc set triggers
will need to support a flag to set and remove this, and the describers will
need to show it.

On Tue, Apr 26, 2016 at 5:40 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/build/api/validation/validation.go
> <https://github.com/openshift/origin/pull/8477#discussion_r61169644>:
>
> > @@ -500,11 +500,14 @@ func validateTrigger(trigger *buildapi.BuildTriggerPolicy, buildFrom *kapi.Objec
> >  	return allErrs
> >  }
> >
> > -func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path) field.ErrorList {
> > +func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path, allowEnv bool) field.ErrorList {
>
> might make more sense to call the allowEnv argument "isGeneric" instead,
> in case the two webhooks evolve further away from each other in the future.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8477/files/c5d7190b4aecda65321f7e58f67bb3f6b2560b5e#r61169644>
>